src/Features/Core/Portable/Shared/TestHooks/Legacy/FeatureAttribute.cs

```diff
@@ -11,11 +11,13 @@ internal partial class FeatureAttribute : Attribute
     {
         private readonly string _featureName;
 
+        [Obsolete("use IAsynchronousOperationListenerProvider")]
```

Comment:
❔ Move to just having it on the type?

Reply:
type is not obsolete. just these methods.